### DISCUSSION TITLE
[提问]我在PCL2里下载Mods / Minecraft的时候遇到了的问题：以一种访问权限不允许的方式做了一个访问套接字的尝试。

### DISCUSSION THREAD

**Discussion by HuanMeng6** [Meloong-Git/PCL#3286] [提问] [answered]
具体症状：
1、无法借助PCL2里的反馈打开反馈界面。提示：你的PCL不是最新版，因此无法提交反馈。
2、无法用龙猫发的更新密钥来更新。
3、在打开下载界面时会提示：以一种访问权限不允许的方式做了一个访问套接字的尝试。 36.147.58.115:443
4、就算重启、更换网络也不行。

**Comment by allMagicNB** [comment]
重装系统

**Comment by SALTWOOD** [comment, marked as answer]
一般来说是端口被占用，但看样子那个IP地址和端口是被访问的，所以大概率是你的电脑的问题（不会是65535个端口都满了？）
还有，以后记得不要在 Issue 里头版聊，Collaborator 的邮箱会炸的（Subscribe 了的也会）

**Comment by HuanMeng6** [comment]
好吧

**Comment by SALTWOOD** [comment]
如图，我用 Python 占满了所有端口，然后所有网站都无法访问
![32c4c9b10bbf8d61899c211c46bbebd6](https://github.com/Hex-Dragon/PCL2/assets/105980161/0cfce0d6-2b18-46bf-bc52-5b37e35ce88e)

你这个样子首先先重启，然后检查有没有异常端口占用，要是没有问题，那就怀疑是系统出问题了，重装或者在同一个网络下另一台电脑上试试；如果还有问题，那就是路由器之类的或者你那台也有问题（
